pep-0497-externals.txt

```diff
@@ -0,0 +1,252 @@
+PEP: 497
+Title: Enhanced handling of external requirements
+Version: $Revision$
+Last-Modified: $Date$
+Author: Tennessee Leeuwenburg <tleeuwenburg@gmail.com>,
+
+Status: Draft
+Type: Standards Track
+Python-Version 3.6
+Content-Type: text/x-rst
+Created: 03-Aug-2015
+Post-History: 03-Aug-2015
+
+
+Abstract
+========
+
+The canonical method for installing packages is to use pip, whether that is
+referencing PyPI or a local repository. A limitation of the current install
+ecosystem is the relationship with operating-system level dependencies. This
+can result in compilation failures or execution failures. Package maintainers
+may wish to specify the requirements to avoid this issue, and users may
+appreciate earlier stopping and more informative messages about their system.
+
+The responsibility of pip is to stop early and provide useful information about
+missing and not automatically installable dependecies. Ideally it would provide
+a useful user message about how to obtain the required dependencies. The
+responsibility for actual installation lies with the user at this time.
+
+This PEP specifies an extension to the requirements sections of setup.py
```

Comment:
Lets be broader here.

This PEP can be thought of as specifying an extension to the descriptions of 'distributions' in PEP-426. I think we need to update that // make this be referenced, and reference PEP-426. https://www.python.org/dev/peps/pep-0426/#name